src/module/settings.ts

```diff
@@ -79,6 +79,20 @@ export const registerSettings = function () {
     default: false,
   });
 
+  game.settings.register(game.system.id, LANCER.setting_square_grid_diagonals, {
+    name: "lancer.squaregriddiagonals.name",
+    hint: "lancer.squaregriddiagonals.hint",
+    scope: "world",
+    config: true,
+    type: String,
+    choices: {
+      "111": "lancer.squaregriddiagonals.111",
+      "121": "lancer.squaregriddiagonals.121",
+      "222": "lancer.squaregriddiagonals.222",
+      "euc": "lancer.squaregriddiagonals.euc",
+    },
+  });
+
```

Comment:
Does this need a `default`? I think v10 requires it but I may be misreading how that works.

Reply:
Yes, but I could have sworn I had it in there.